cli/cluster/membership.go

```diff
@@ -24,24 +24,34 @@ func (membershipInstance *MembershipInstance) DecodeMsgpack(d *msgpack.Decoder)
 	return common.DecodeMsgpackStruct(d, membershipInstance)
 }
 
+func GetFirstInstanceName(instances map[string]string) (string, bool) {
+	for _, instanceName := range instances {
+		return instanceName, true
+	}
+	return "", false
+}
+
+type InstancesFilter func(*MembershipInstances, map[string]string)
```

Comment:
Probably, move type definition to the begining of the file. Also you can add a description to it.